spec/lib/formatter_spec.rb

```diff
@@ -126,8 +126,8 @@ module CodeClimate::TestReporter
         end
 
         it 'should include the path prefix if set' do
-          expect(formatter.short_filename('file1')).to eq('custom/file1')
-          expect(formatter.short_filename("#{::SimpleCov.root}/file1")).to eq('custom/file1')
+          expect(formatter.send(:short_filename, 'file1')).to eq('custom/file1')
+          expect(formatter.send(:short_filename, "#{::SimpleCov.root}/file1")).to eq('custom/file1')
```

Comment:
I happen to agree with you and didn't intend for you to do any re-architecture. Just wanted to add the least amount of code to get the job done. Thanks for your contribution.